sklearn/svm/base.py

```diff
@@ -111,7 +111,7 @@ def fit(self, X, y, sample_weight=None):
 
         Parameters
         ----------
-        X : {array-like, sparse matrix}, shape (n_samples, n_features)
+        X : {array-like, sparse matrix}, shape = [n_samples, n_features]
```

Comment:
Do you have a strong reason to replace parentheses by square brackets. I am not sure there is a strong consistency in the code base but I would personally favor parentheses since this looks like a tuple which is the type of `X.shape`.

I could not find anything about how to document shape in https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt but please share a link to best practices if you have one.

Reply:
A not very well-check `git grep` popularity contest seems to indicate that parentheses are twice as more popular in the scikit-learn code:

```
❯ git grep -P '\w+ : .+shape' | grep -P 'shape[ =]*+\[' | wc -l
765

❯ git grep -P '\w+ : .+shape' | grep -P 'shape[ =]*+\(' | wc -l
1530
```

Reply:
Do I use parentheses for the docstring? I just updated them assuming that the result is returned in an array form.

Reply:
Shape is a tuple, technically, so parentheses are appropriate

Reply:
this style:

```
{array-like, sparse matrix}, shape (n_samples, n_features)
```

is the numpy recommended way. We started with 

```
{array-like, sparse matrix}, shape = [n_samples, n_features]
```

and if it's still like this it's for historical reasons. let's not move backwards here.